app/overlays/mtl/dax_overlay.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_COMP_MODULE_ADAPTER=y
+CONFIG_COMP_DOLBY_DAX_AUDIO_PROCESSING=y
+CONFIG_COMP_DOLBY_DAX_AUDIO_PROCESSING_MOCK=n
+CONFIG_KCPS_DYNAMIC_CLOCK_CONTROL=n
```

Comment:
Just a question - any reason why it was disabled in dax overlay? Is there a problem with KCPS and DAX module?